.changes/next-release/Bug Fix-96a391a0-ee15-4ccb-a750-7d02fdaa8d1a.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Can you split `All arrow functions in Mocha tests were changed to standard function declarations` into it's own PR? It's difficult to find the other changes since every test file is changed

Reply:
I'll close this PR and split it into 3 new ones